src/components/TextArea/TextArea.driver.ts

```diff
@@ -75,5 +77,8 @@ export const textAreaDriverFactory = (base: UniDriver): TextAreaDriver => {
     async success() {
       return isSuccess(base);
     },
+    async hasAttribute(attr: string) {
+      return (await textArea(base).attr(attr)).length > 0;
+    },
```

Comment:
I don't think we need this method.
I think you're not using it, and it's very generic, and in any case consumers using the driver can do:
```js
(await driver.element()).getAttribute('my attribute');
```

Also, just for future reference, this could have been written like this:
```js
    async hasAttribute(attr: string) {
      return !!(await base.attr(attr));
    },
```

Reply:
Yeah sorry about this, I didn't meant to commit it.